index.js

```diff
@@ -214,27 +214,40 @@ function init (ssb, config) {
    * (because they can't post messages to the group before then)
    */
 
+  const memberType = (type) => type === 'group/add-member' || type === 'group/exclude-member'
```

Comment:
rename `isMemberType` to hint it will return `Boolean`

Reply:
done